pom.xml

```diff
@@ -422,7 +423,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
-        <version>4.3.1</version>
+        <version>${httpclient.version}</version>
```

Comment:
We were actually using both.  It looks `org.apache.httpcomponents:httpclient` is newer so I will see if we can only use that.